src/app/shared/pipes/translation-known-values.pipe.ts

```diff
@@ -0,0 +1,34 @@
+import { Pipe, PipeTransform } from '@angular/core'
+import { Tv2ActionContentType } from '../models/tv2-action'
+
+@Pipe({
+  name: 'translationKnownValues',
+})
+export class TranslationKnownValuesPipe implements PipeTransform {
+  public transform(value: string): string {
```

Comment:
Since we know that we are translating `Tv2ActionContentType` then the parameter should be `contentType: Tv2ActionContentType`